locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -78,7 +89,7 @@ export const notifyNewKeysToWedlocks = async (
  * and email based on the lock's template if applicable
  * @param key
  */
-export const notifyNewKeyToWedlocks = async (key: any, network?: number) => {
+export const notifyNewKeyToWedlocks = async (key: Key, network?: number) => {
```

Comment:
Let's make sending the QR code optional here so that the background job _does not_ send the QR code, but the explicit API call does. Does it make sense?